site/_contributors/gene.txt

```diff
@@ -0,0 +1,6 @@
+---
+weight: 10
```

Comment:
:P :P I apologize! I'll open an issue to make sure this is changed to `position` (I agree it makes more sense) + explain it clearly in the contributing doc